frontend/src/App.tsx

```diff
@@ -378,64 +377,6 @@ export class App extends PureComponent<Props, State> {
     })
   }
 
-  /**
-   * Handler for ForwardMsg.initialize messages
-   * @param initializeMsg an Initialize protobuf
-   */
-  handleInitialize = (initializeMsg: Initialize): void => {
-    const {
-      sessionId,
-      environmentInfo,
-      userInfo,
-      config,
-      sessionState,
-    } = initializeMsg
-
-    if (
-      sessionId == null ||
-      !environmentInfo ||
-      !userInfo ||
-      !config ||
-      !sessionState
-    ) {
-      throw new Error("InitializeMsg is missing a required field")
```

Comment:
I'm assuming this part isn't necessary now. Are we just assuming the correct message is sent down (totally fine, but just checking)?

Reply:
All these checks happen within the SessionInfo constructor, so this was just essentially duplicate logic